src/plugins/ogr2ogr_operator.py

```diff
@@ -125,7 +125,7 @@ def execute(self, context: Optional[Dict[str, Any]] = None) -> None:
 
         # generic optional parameters
         if self.sql_statement:
-            ogr2ogr_cmd.append(f"-sql {self.sql_statement} ")
+            ogr2ogr_cmd.append(f'-sql "{self.sql_statement}" ')
```

Comment:
Since this is going to a shell, double quotes will cause expansion of environment variables. Do we need that? If not, can we do

    ogr2ogr_cmd.append(f"-sql '{self.sql_statement}' ")

here? Even better, can we get rid of `shell=True`?

Reply:
Good catch. Will change it.

Reply:
Done

Reply:
Push?